app/views/org_admin/templates/_templates_list.html.erb

```diff
@@ -0,0 +1,66 @@
+<% # locals: templates, current_org %>
+
+<div class="table-responsive">
+  <table class="table table-hover table-bordered tablesorter">
+    <thead>
+      <% if scope.length > TABLE_FILTER_MIN_ROWS %>
+        <tr>
+          <th colspan="5" class="sorter-false">
+            <%= render(partial: "shared/table_filter", locals: { placeholder: _('Search')}) %>
+          </th>
+        </tr>
+      <% end %>
+      <tr>
+        <th><%= _('Template Name') %></th>
+        <th><%= current_user.can_super_admin? ? _('Organisation') : _('Description') %></th>
+        <th><%= _('Status') %></th>
+        <th><%= _('Edited Date') %></th>
+        <th class="sorter-false"></th>
+      </tr>
+    </thead>
+    <tbody>
+    <% scope.each do |template| %>
+      <tr>
+        <td>
+          <%= template.title %>
+        </td>
+          <td>
+            <%= current_user.can_super_admin? ? template.org.name : raw(template.description) %>
+          </td>
+          <td>
+            <%= (template.published? ? _('Published') : _('Unpublished')) %>
+          </td>
+          <td>
+            <% last_temp_updated = template.updated_at %>
+            <%= l last_temp_updated.to_date, formats: :short %>
+          </td>
+          <td>
+            <div class="dropdown<%= current_org == template.org.id ? '' : ' hide'%>">
```

Comment:
Unless we are un-hiding this from the JS, I propose we just don't render a dropdown menu for un-editable templates.

Reply:
invalidated by another comment